packages/xo-server-sdn-controller/src/ovsdb-client.js

```diff
@@ -110,7 +112,14 @@ export class OvsdbClient {
     const portName = bridgeName + '_port' + index
 
     // Add interface and port to the bridge
-    const options = ['map', [['remote_ip', remoteAddress], ['key', key]]]
+    const options =
+      password !== undefined
+        ? [
+            'map',
+            [['remote_ip', remoteAddress], ['key', key], ['psk', password]],
+          ]
+        : ['map', [['remote_ip', remoteAddress], ['key', key]]]
```

Comment:
Wouldn't it be easier to read if you declared `options` first and then pusk `psk` into it if `password` is defined?

Reply:
Yes, wayyyyyyyyyyyy better!